Homework9_QueueAndStack/QueueAndStackTasks.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Homework9_QueueAndStack
+{
+    class QueueAndStackTasks : QueueAndStackMethods
+    {
+        private const int ListLength = 7;
+        private const int NumberOfChars = 3;
+
+        private static void Main(string[] args)
```

Comment:
Common for all files. Main method should be public, other methods can be private

Reply:
fixed